app/models/null_user.rb

```diff
@@ -0,0 +1,17 @@
+class NullUser
+  def is_admin?
+    false
+  end
+
+  def is_manager?
+    false
+  end
+
+  def is_viewer?
```

Comment:
Addressed now in an updated version of https://github.com/sul-dlss/argo/commit/6137adf19318612dec03e4e5ba850b65f4deddab#diff-3d4dc98d40fd4202d526f609441144dbR2